src/adapters/sqlite/index.js

```diff
@@ -157,8 +162,10 @@ export default class SQLiteAdapter implements DatabaseAdapter, SQLDatabaseAdapte
           ),
         )
         logger.log('[WatermelonDB][SQLite] Migration successful')
+        this.migrationEvents?.onSuccess?.()
       } catch (error) {
         logger.error('[WatermelonDB][SQLite] Migration failed', error)
+        this.migrationEvents?.onFailure?.(error)
```

Comment:
sorry, but no `?.(` syntax please - it breaks some of our tooling

Reply:
corrected.